k8s/sandbox/cluster-01-overlay/kustomization.yaml

```diff
@@ -33,6 +33,9 @@ patchesStrategicMerge:
   #kured patch
   - ../../namespaces/kured/patches/sandbox/cluster-01/kured.yaml
 
+  #aad-pod-identity patch
```

Comment:
now, i am thinking this patch should just go into base kustomize ? as its same across all clusters

Reply:
resolved